lua/search/init.lua

```diff
@@ -202,6 +215,7 @@ M.open = function(opts)
 	M.reset(opts)
 	M.opened_on_win = vim.api.nvim_get_current_win()
 	M.busy = true
+	M.opened_from_buffer = true
```

Comment:
Suggestions for a better name than this?
It is used to indicate if the telescope window was launched from a buffer instead of launched through tab navigation.

Maybe is_tab_switch?

Reply:
Maybe is_initial_tab? Could be mistaken with if it is the tab defined as initial tho... 
opened_from_buffer is kind of misleading, since every tab is actually a window but I'm fine with leaving it like this :smile: